src/lib/toastr/toastr-config.ts

```diff
@@ -97,23 +97,23 @@ export interface GlobalConfig extends IndividualConfig {
    * Zero is unlimited
    * default: 0
    */
-  maxOpened?: number;
+  maxOpened: number;
   /**
    * dismiss current toast when max is reached
    * default: false
    */
-  autoDismiss?: boolean;
-  iconClasses?: ToastrIconClasses;
+  autoDismiss: boolean;
+  iconClasses: Partial<ToastrIconClasses>;
```

Comment:
I don't like doing this but otherwise there isn't a way to individually override iconClasses